mkdocs.yml

```diff
@@ -97,6 +97,8 @@ nav:
           - Parallel Processing Python: _user_guide/best_practices_parallel_processing_python.md
       - R:
           - Best Practices in R: _user_guide/best_practices_r.md
+      - Matlab:
+          - Matlab Slurm Jobs:  _user_guide/best_practices_matlab_slurm_job.md
```

Comment:
Change name of file from "_user_guide/best_practices_matlab_slurm_job.md" to "_user_guide/matlab-slurm.md". I saw this change in [Natalya's Julia Slurm PR](https://github.com/gsbdarc/rcpedia_mockup/pull/55/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2).

Reply:
Should it be matlab_slurm.md since all other files use _ conventions and not - ? @mpj2104

Reply:
I dont have a strong preference for - vs _, mainly removing the "best_practices_" part based on what I saw Natalya do